### DISCUSSION TITLE
Proxy through DevServer/Bundler API

### DISCUSSION THREAD

**Discussion by tascord** [parcel-bundler/parcel#8437] [Q&A]
Hey!

Is it possible to specify either directly, or specify a file, that the bundler/devserver will use instead of the .proxyrc file?

Currently I have a setup like this
```ts
const bundler = new Parcel({
    entries: [
        relative_path('../test/material/1.tsx')
    ],
    config: relative_path('../resources/parcel_config.json'),
    mode: 'development',
    serveOptions: { port: 3000 },
    hmrOptions: { port: 3000 },
    logLevel: 'verbose',
    additionalReporters: [
        // todo: add custom reporter
        {
            packageName: '@parcel/reporter-cli',
            resolveFrom: fileURLToPath(import.meta.url),
        }
    ]
})
```

but due to the nature of this project, all of the paths are quite messy.

**Comment by mischnic** [comment]
Unfortunately not, it always reads from the config file:
https://github.com/parcel-bundler/parcel/blob/527e477d15441219216ed05b220cf8d48fa2d258/packages/reporters/dev-server/src/Server.js#L384-L429
